spec/index.bs

```diff
@@ -4215,48 +4215,58 @@ and {{GPUCommandEncoder/copyTextureToBuffer()}}.
 
 Issue: define this as an algorithm with (texture, mipmapLevel) parameters and use the call syntax instead of referring to the definition by label.
 
-<div algorithm class=validusage>
+<div algorithm>
```

Comment:
The spec flips back and forth between declaring algorithms as functions like this and declaring them with prose (like https://gpuweb.github.io/gpuweb/#abstract-opdef-the-steps-to-serialize-a-gpudevice-object). Feels like we should probably pick one style and stick with it?

FWIW I don't have much in the way of opinons about which to prefer. For more complex algorithms this feels a bit cleaner, though possibly easy to confuse as an API-facing method at first glance. For smaller algorithms it feels overly bulky.

Reply:
I think it's probably okay to have a mix of both depending on what fits the case best. I don't think confusing with API facing methods will be too much of a problem: those are nested in `<dl>` and use `argumentdef`. But they have gotten fairly similar.

In the case of the serialization algorithms in particular, we don't define the types of the arguments currently (they're implicitly the same as in the HTML spec link right next to them) but we could use this newer format.